testing/tests/DevExpress.ui.widgets.dataGrid/gridView.tests.js

```diff
@@ -2225,8 +2225,8 @@ QUnit.module('Fixed columns', {
 
         // assert
         assert.equal($colElements.length, 3, 'count col');
-        assert.notOk($colElements.eq(0).attr('style'), 'width of the first col');
-        assert.notOk($colElements.eq(1).attr('style'), 'width of the second col');
+        assert.strictEqual($colElements.eq(0).attr('style'), 'width: auto;', 'width of the first col');
+        assert.strictEqual($colElements.eq(1).attr('style'), 'width: auto;', 'width of the second col');
         assert.strictEqual($colElements.eq(2).attr('style'), 'width: 100px;', 'width of the third col');
```

Comment:
this test is checking that in fixed table's colgroup third column has 'width: 100px' if there is no scroll. Widths of first two columns are not important.